components/atoms/Input.tsx

```diff
@@ -0,0 +1,22 @@
+import { styled } from '@/stitches.config';
+import { PropsWithChildren } from 'react';
+
+export interface InputProps {
```

Comment:
What is the purpose of this interface? since are creating a styled component and a React function doesn't need to create an interface

Reply:
Same here, I had declared that interface during the initial implementation of the component before I went and implemented the component using the Stitches `styled()` function. Forgot to clean that up 😜